Hunting Queries/SecurityEvent/KrbRelayUpServiceCreation

```diff
@@ -0,0 +1,38 @@
+id: 06f83b36-a1df-4045-98a5-deda74d84e4a
+name: KrbRelayUp Local Privilege Escalation Service Creation
+description: |
+   'This query detects the default service name created by KrbRelayUp. KrbRelayUp is Local Privilege Escalation tool that combine features of Rubeus and KrbRelay.
+severity: High
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - Event
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1543
+query: |
+  let MaliciousService = dynamic (["KrbSCM"]);
+  Event
+  | where Source == "Service Control Manager" and EventID == 7045
+  | parse EventData with * 'ServiceName">' ServiceName "<" * 'ImagePath">' ImagePath "<" *
+  | where ServiceName has_any (MaliciousServiceArtifacts) or ImagePath has_any (MaliciousServiceArtifacts)
```

Comment:
Seems like we should have MaliciousService vs  MaliciousServiceArtifacts in here.